graceoncampus/goc-web-v3
# Add styling to rider signup form + common styling sass file

Testing Steps
------------------------------------------------------------------------
1. yarn && yarn build && yarn start
2. navigate to localhost:3000
3. ensure that the landing page matches up to the mock based on changes

Resources
------------------------------------------------------------------------
- sass: https://sass-lang.com/
- sass-rem (npm package added): https://www.npmjs.com/package/sass-rem
- react-bootstrap: https://react-bootstrap.github.io/
